visualisation/3dSDPViewer.cpp

```diff
@@ -428,19 +428,16 @@ int main( int argc, char** argv )
     }
     if(vm.count("addMesh"))
     {
-      DGtal::Color meshColor = DGtal::Color::White;
-      if(vm.count("customColorMesh")){
+      bool customColorMesh =  vm.count("customColorMesh");
```

Comment:
for my part I prefer shorter, but in fact we have the same in other tools I think, perhaps we could change it globaly  if needed ;)